src/exts/docs.py

```diff
@@ -0,0 +1,55 @@
+from urllib.parse import quote
```

Comment:
`quote_plus` is probably what you're looking for, not `quote`. `quote`, among other things, doesn't handle spaces.

Reply:
I tried quote_plus, but seems like quote is what readthedocs want

Reply:
...huh.